Python/Product/Cookiecutter/CookiecutterToolWindow.cs

```diff
@@ -209,21 +209,19 @@ sealed class CookiecutterToolWindow : ToolWindowPane, IVsInfoBarUIEvents {
             _uiShell.UpdateCommandUI(0);
         }
 
-        private void OpenGeneratedFolder(string folderPath) {
-#if DEV15_OR_LATER
-            OpenInSolutionExplorer(folderPath);
-#else
-            OpenInWindowsExplorer(folderPath);
+        private void ExecuteCommand(string name, string args) {
+            try {
+                _dte.ExecuteCommand(name, args ?? string.Empty);
```

Comment:
We should be using `IVsUIShell.PostExecCommand` here with a GUID and ID. Using strings is going to break on non-English VS, as the names are localised. See ImportWizard.Wizard.RunStarted for an example (including passing parameters).